### DISCUSSION TITLE
Is there a migration guide from Vitest 0.25.7 to the current Vitest version?

### DISCUSSION THREAD

**Discussion by MikeJakuszewski** [vitest-dev/vitest#5144] [General]
I need to update this dependency to the latest version. The only guide I've found is for 0.34.6, would this work for older versions?
https://vitest.dev/guide/migration.html#migrating-from-vitest-0-34-6

**Comment by AriPerkkio** [comment]
There are no migration guides for upgrading from `0.x.x` to another `0.x.x` release. However you should find all breaking changes from the releases page https://github.com/vitest-dev/vitest/releases.

**Reply by MikeJakuszewski** [reply]
I'm just looking to go from 0.25.7 to the latest version. Are there any docs that show how to do that from 0.25.7?
